### DISCUSSION TITLE
exchanged the samples in 1 directory

### DISCUSSION THREAD

**Discussion by streetuff** [copych/AcidBox#18] [Show and tell]
here https://github.com/streetuff/acid-box-samples are new samples for the data/1 directory i sampled from my roland tr6-s. maybe it's usefull for someone. has a quite deep bassdrum in it. could be exchanged for other directorys just rename the files 101/102/103/etc. to the directory. like 701/702/703/etc....

**Comment by copych** [comment]
Hello and thank you for the contribution @streetuff ! Actually you don't have to rename each file to match the containing directory. Renaming the dir is enough, they won't confuse. You can delete or add folders with samples, the limitation here is LittleFS flash partition size.
If you'd like, I'll make a link in the readme to your repo, or whatever you suggest.
